binding/SkiaSharp.NativeAssets.WebAssembly/buildTransitive/SkiaSharp.targets

```diff
@@ -14,15 +19,25 @@
   </ItemGroup>
 
   <!-- If this is a ASP.NET Blazor web assembly app -->
-  <PropertyGroup Condition="'$(UsingMicrosoftNETSdkBlazorWebAssembly)' == 'true'">
+  <PropertyGroup Condition="'$(UsingMicrosoftNETSdkBlazorWebAssembly)' == 'true' and '$(ShouldIncludeNativeSkiaSharp)' == 'True'">
     <WasmBuildNative Condition="'$(WasmBuildNative)' == ''">true</WasmBuildNative>
     <EmccExtraLDFlags>$(EmccExtraLDFlags) -s USE_WEBGL2=1</EmccExtraLDFlags>
```

Comment:
Note to me: I see Uno has `-s OFFSCREEN_FRAMEBUFFER=1` in the flags, so I am wondering if we need that too for Blazor (I was not able to test before since this feature did not exist yet). However, this probably means it is actually needed so maybe I need to check that as well.